### ISSUE TITLE
DM_SERVER_E_INI_FILE_ACCESS error

### ISSUE THREAD

**Issue by karolbe** [jppop/dctm-docker#4]
Hello

I can't get it working, it fails in dctm-cs with the error as shown below. Any ideas what might be wrong? I am installing CS 7.1. 

done
Stopping the repository
/opt/documentum/entrypoint.sh: line 136: /opt/documentum/dba/dm_shutdown_devbox: No such file or directory
Starting the repository devbox..
- starting jms in background..
- starting repoditory server..
  2015-12-05T00:11:33.915409  179[179]    0000000000000000    [DM_SERVER_E_INI_FILE_ACCESS]error:  "Error initializing process due to the inability to get startup parameters from the named init_file: /opt/documentum/dba/config/devbox/server.ini.  Error reason:  errno: 2, message: No such file or directory."

Usage: ./documentum -docbase_name <docbase name> -init_file <filename> [-o<option>]

```
-docbase_name : name of docbase
-init_file    : name of server.ini file (including path) which
                contains the server startup parameters
```

Usage: ./documentum -h

**Comment by jppop** [comment]
Hello,

Could be disk space issue.
We use host machine (where docker is running) with 16GB RAM and a disk with at least 80 GO.

**Comment by karolbe** [comment]
Actually, this looks like a duplicate of the closed issue #3 , I have following error in the log:

00:27:35,429 INFO [main] com.documentum.install.server.installanywhere.actions.DiWAServerCheckKeystoreStatus - The installer detected the keystore already exists and was created using default password.
00:27:37,434 INFO [main] com.documentum.install.server.installanywhere.actions.DiWAServerQueryDatabaseInformation - The installer is gathering database connection information from the local machine.
00:27:37,441 INFO [main] com.documentum.install.appserver.jboss.JbossApplicationServer - setJboss sharedDfcLibDir is:/opt/documentum/shared/dfc
00:27:37,441 INFO [main] com.documentum.install.appserver.jboss.JbossApplicationServer - getFileFromResource for templates/appserver.properties
00:27:37,443 INFO [main] com.documentum.install.server.installanywhere.actions.DiWAServerUpdateTCSUnixServiceFile - The installer will check service entries for repository devbox.
00:27:39,449 INFO [main] com.documentum.install.server.installanywhere.actions.DiWAServerLoadServerWebcacheInfo - The installer will obtain database information of XE.
00:27:39,450 INFO [main] com.documentum.install.server.installanywhere.actions.DiWAServerTestDatabaseConnection - The installer is testing the database connection information
00:27:39,451 INFO [main] com.documentum.install.server.common.services.db.DiServerDbSvrOracleServer - The installer is testing the database connectivity for the user "system".
00:27:39,593 INFO [main] com.documentum.install.server.common.services.db.DiServerDbSvrOracleServer - The installer is validating the database version is supported.
00:27:39,710 INFO [main] com.documentum.install.server.common.services.db.DiServerDbSvrOracleServer - The installer is checking for the database TEMP tablespace.
00:27:39,830 INFO [main] com.documentum.install.server.installanywhere.actions.DiWAServerTestDocbaseOwnership - The installer is verifying that there is no repository owned by devbox.
00:27:40,018 INFO [main] com.documentum.install.server.installanywhere.actions.DiWAServerQueryTablespaceName - The installer is gathering all accessible tablespace names from the selected database. 
00:27:40,212 INFO [main] com.documentum.install.server.installanywhere.actions.DiWAServerObtainOracleDataFileDirectory - The installer is locating the default Oracle data file directory.
00:27:40,401 INFO [main] com.documentum.install.server.installanywhere.actions.DiWAServerLoadDataFileInfo - The installer will set default database information for repository devbox.
00:27:40,405 INFO [main] com.documentum.install.server.installanywhere.actions.DiWAServerCheckProcessXHive - processing xhive=false
00:27:40,408 INFO [main] com.documentum.install.server.installanywhere.actions.DiWAServerModifyDfcProperties - The installer will update dfc.properties file.
00:27:40,410 INFO [main] com.documentum.install.shared.common.services.dfc.DiDfcProperties - Installer is adding it as primary connection broker and moves existing primary as backup.
00:27:40,414 ERROR [main] com.documentum.install.server.installanywhere.actions.DiWAServerModifyDfcProperties - Failed to backup file: /opt/documentum/shared/config/dfc.properties - current user may lack adequate security access privileges. Click Yes to continue,click NO to exit from the installer.
com.documentum.install.shared.common.error.DiException: Failed to backup file: /opt/documentum/shared/config/dfc.properties - current user may lack adequate security access privileges.
at com.documentum.install.shared.common.services.DiUtil.backupFile(DiUtil.java:1866)

**Comment by karolbe** [comment]
It was a problem with access to /opt/documentum/shared/config/dfc.properties. I accidentally commented out the "USER dmadmin"  line from dctm-base/Dockerfile:85 because I didn't want to install the patch and dfc.properties was copied with root user...

Now all is working fine! :-)
